python_ta/transforms/type_inference_visitor.py

```diff
@@ -352,19 +352,23 @@ def lookup_type(self, node, name) -> type:
     ##############################################################################
     @accept_failable
     def get_call_signature(self, c: type, node: NodeNG) -> TypeResult:
-        if hasattr(c, '__name__') and c.__name__ == 'Type':
+        # Callable type; e.g., 'Callable[[int], int]'
+        if isinstance(c, CallableMeta):
+            return TypeInfo(c)
+        # Union of Callables
+        elif c.__class__.__name__ == '_Union' and all(isinstance(elt, CallableMeta) for elt in c.__args__):
+            return TypeInfo(c)
+        # Class types; e.g., 'Type[_ForwardRef('A')]'
+        elif hasattr(c, '__name__') and c.__name__ == 'Type':
```

Comment:
Ah, I forgot to mention this earlier. Use `getattr` instead of this pattern (default returns `None` I think).